src/www/ui_components/language-view.html

```diff
@@ -0,0 +1,112 @@
+<!--
+  Copyright 2018 The Outline Authors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<link rel="import" href="../bower_components/polymer/polymer-element.html" />
+<link rel="import" href="../bower_components/polymer/lib/mixins/dir-mixin.html" />
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-repeat.html" />
+<link rel="import" href="../bower_components/polymer/lib/elements/dom-if.html" />
+
+<dom-module id="language-view">
+  <template>
+    <style>
+      :host {
+        background: #fff;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        text-align: center;
+        width: 100%;
+        height: 100vh;
+      }
+      .language-item {
+        display: flex;
+        cursor: pointer;
+        font-size: 16px;
+        border-bottom: 1px solid #e0e0e0;
+        padding-left: 24px;
+        --paper-item-selected: {
+          color: var(--medium-green);
+          font-weight: normal;
+        }
+      }
+      .language-name {
+        text-align: left;
+        flex-grow: 1;
+      }
+    </style>
+
+    <div id="main">
+      <paper-listbox selected="{{selectedLanguage}}" attr-for-selected="value" on-selected-changed="_languageSelected">
+        <template is="dom-repeat" items="{{languages}}" as="lang" sort="_sortLanguageNameAscending">
+          <paper-item class="language-item" value="{{lang.id}}">
+            <span class="language-name">{{lang.name}}</span>
+            <iron-icon icon="check" hidden$="{{_shouldHideCheck(selectedLanguage, lang.id)}}"></iron-icon>
+          </paper-item>
+        </template>
+      </paper-listbox>
+    </div>
+  </template>
+
+  <script>
+    class OutlineLanguageView extends Polymer.DirMixin(Polymer.Element) {
+      static get is() {
+        return "language-view";
+      }
+
+      static get properties() {
+        return {
+          selectedLanguage: {
```

Comment:
nit: `selectedLanguage: String` (if you don't need to set other attributes)

Reply:
Done